src/Patches/PlayerCharacterPatches.cs

```diff
@@ -352,6 +352,11 @@ public class PlayerCharacterPatches {
             if (PaletteEditor.PartyHatEnabled) {
                 WearHat = true;
             }
+            // make it so you can pick up money from further away
+            List<ItemPickup> coins = Resources.FindObjectsOfTypeAll<ItemPickup>().Where(coin => coin.gameObject.scene.name == "DontDestroyOnLoad").ToList();
+            foreach (ItemPickup coin in coins) {
+                coin.minimumAttractDistance = 6.5f;
+            }
```

Comment:
Let's move this to ModelSwaps.InitializeItems() or somewhere similar, that way it only ever gets called once and never again